controller/coq_lsp.ml

```diff
@@ -76,10 +76,12 @@ let do_client_options coq_lsp_options =
 
 let check_client_version version : unit =
   LIO.trace "version" version;
-  match version with
-  | "any" | "0.1.3" -> ()
-  | v ->
-    let message = Format.asprintf "Incorrect version: %s , expected 0.1.2" v in
+  if Version.equal (Version.of_string version) Version.current then ()
```

Comment:
PR looks good to me thanks, tho I'm unsure if this whole dance `of_ / to_` is worth it.

If I was reading this, I'd be confused about the semantics of `Version.equal`, IMVHO older code was more explicit.